src/java.base/share/classes/sun/net/www/protocol/https/HttpsClient.java

```diff
@@ -565,6 +566,7 @@ public void afterConnect() throws IOException, UnknownHostException {
                     // will do the spoof checks in SSLSocket.
                     SSLParameters paramaters = s.getSSLParameters();
                     paramaters.setEndpointIdentificationAlgorithm("HTTPS");
+                    paramaters.setServerNames(Collections.singletonList(new SNIHostName(host)));
```

Comment:
List.of() is more preferred now.

Is the code in line 465 and 467 is duplicated now?

Reply:
used List.of(), but i a not able to follow line 465 and 467. Are we are looking same code base repo?